src/px-script/main/utils/index.js

```diff
@@ -30,8 +30,28 @@ export const handleScrollbar = () => {
     }
 };
 
+export const findElemsWithId = (ids, elementName) => {
+    const elements = [];
+    const pushElement = id => {
+        const val = document.querySelector(`#${id}`);
```

Comment:
If you are only getting elements by id's, why not use `document.getElementById`?